sdk/privatedns/Azure.ResourceManager.PrivateDns/api/Azure.ResourceManager.PrivateDns.netstandard2.0.cs

```diff
@@ -377,36 +421,65 @@ public partial class VirtualNetworkLinkResource : Azure.ResourceManager.ArmResou
+        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.PrivateDns.VirtualNetworkLinkResource>> UpdateAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.PrivateDns.VirtualNetworkLinkData data, Azure.ETag? ifMatch = default(Azure.ETag?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
 namespace Azure.ResourceManager.PrivateDns.Models
 {
-    public partial class AaaaRecord
+    public partial class PrivateDnsAaaaRecordInfo
     {
-        public AaaaRecord() { }
-        public string IPv6Address { get { throw null; } set { } }
+        public PrivateDnsAaaaRecordInfo() { }
+        public System.Net.IPAddress IPv6Address { get { throw null; } set { } }
     }
-    public partial class ARecord
+    public partial class PrivateDnsARecordInfo
     {
-        public ARecord() { }
-        public string IPv4Address { get { throw null; } set { } }
+        public PrivateDnsARecordInfo() { }
+        public System.Net.IPAddress IPv4Address { get { throw null; } set { } }
     }
-    public partial class MxRecord
+    public partial class PrivateDnsMXRecordInfo
     {
-        public MxRecord() { }
+        public PrivateDnsMXRecordInfo() { }
         public string Exchange { get { throw null; } set { } }
         public int? Preference { get { throw null; } set { } }
     }
+    public partial class PrivateDnsPtrRecordInfo
+    {
+        public PrivateDnsPtrRecordInfo() { }
+        public string PtrDomainName { get { throw null; } set { } }
+    }
+    public partial class PrivateDnsSoaRecordInfo
+    {
+        public PrivateDnsSoaRecordInfo() { }
+        public string Email { get { throw null; } set { } }
+        public long? ExpireTimeInSeconds { get { throw null; } set { } }
+        public string Host { get { throw null; } set { } }
+        public long? MinimumTtlInSeconds { get { throw null; } set { } }
+        public long? RefreshTimeInSeconds { get { throw null; } set { } }
+        public long? RetryTimeInSeconds { get { throw null; } set { } }
+        public long? SerialNumber { get { throw null; } set { } }
+    }
+    public partial class PrivateDnsSrvRecordInfo
+    {
+        public PrivateDnsSrvRecordInfo() { }
+        public int? Port { get { throw null; } set { } }
+        public int? Priority { get { throw null; } set { } }
+        public string Target { get { throw null; } set { } }
+        public int? Weight { get { throw null; } set { } }
+    }
+    public partial class PrivateDnsTxtRecordInfo
+    {
+        public PrivateDnsTxtRecordInfo() { }
+        public System.Collections.Generic.IList<string> Values { get { throw null; } }
+    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct ProvisioningState : System.IEquatable<Azure.ResourceManager.PrivateDns.Models.ProvisioningState>
```

Comment:
Too generic, we should rename this model

Reply:
`ProvisioningState` rename to `PrivateDnsProvisioningState `